### PR TITLE
Add camera movement

### PR DESCRIPTION
**Pull request by Westlanderz**
This PR adds the movement system for the camera by making use of a camera controller and movement controller. With this you can control the camera through the scene with WASD.

### CODE REVIEW

File: Assets/Scripts/CameraController.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+using UnityEngine;
+
+class CameraController : MonoBehaviour {
+    // The movmentcontroller of the camera
```
Comment (Dennisbonke) [inline Assets/Scripts/CameraController.cs:4]: A typo here, it should be movementcontroller, not movmentcontroller.

File: Assets/Scripts/CameraMovement.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        camera_ = camera;
+    }
+
+
+    /*
+     * This handles the input for the camera and moves the camera with the appropriate vector.
+     * This is called every frame.
+     */
+    public void HandleInput() {
+        if (Input.GetKey(KeyCode.W)) {
```
Comment (Dennisbonke) [inline Assets/Scripts/CameraMovement.cs:20]: This doesn't follow the coding style, `if` statements do not get a space before the opening `(`.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Code looks good, I assume you tested it of course. A few notes on the coding style tho.

- The script files seem to be space indented, we use tabs.
- Comments are full English sentences, they start with a capital and end with a `.`, please recheck them.
- All files (where applicable of course) shall have an empty line at the end of the file, which is lacking here too.

**Comment by Westlanderz** [comment]
@Dennisbonke please check the new commit for the mistakes that should have been fixed now

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**